benchmarks/bench_20newsgroups.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function, division
+from __future__ import division
```

Comment:
I think the scope of this PR is removing all `__future__` imports :)

Reply:
for sure i was having some errors removing these , i ll get back to you on removing all then :+1: